EvidenceApi.Tests/V1/E2ETests/DocumentSubmissionsTest.cs

```diff
@@ -132,6 +132,87 @@ public async Task UnsuccessfulWithInvalidParams()
             response.StatusCode.Should().Be(400);
         }
 
+        [Test]
+        public async Task CreateDocumentSubmissionUnsuccessfulWhenDocumentApiCannotCreateClaim()
```

Comment:
I think test names should be general and not depend on the underlying functionality/system. In you still want to go ahead with this name:
```suggestion
        public async Task CreateDocumentSubmissionUnsuccessfulWhenDocumentsApiCannotCreateClaim()
```